gui/public/i18n/de/translation.ftl

```diff
@@ -487,33 +507,33 @@ onboarding-automatic_proportions-put_trackers_on-title = Legen Sie Ihre Tracker
 onboarding-automatic_proportions-put_trackers_on-description = Um Ihre Proportionen zu kalibrieren, werden wir die Tracker verwenden, die Sie gerade zugewiesen haben. Legen Sie alle Ihre Tracker an. Sie können rechts in der Abbildung sehen, welche welche sind.
 onboarding-automatic_proportions-put_trackers_on-next = Ich habe alle meine Tracker angelegt
 onboarding-automatic_proportions-preparation-title = Vorbereitung
-onboarding-automatic_proportions-preparation-description = Stellen Sie einen Stuhl direkt hinter Ihnen in Ihrem Spielbereich. Seien Sie bereit, während der Autobone-Einrichtung zu sitzen.
+onboarding-automatic_proportions-preparation-description = Stellen Sie einen Stuhl direkt hinter in Ihren Spielbereich. Seien Sie bereit, während der Autobone-Einrichtung zu sitzen.
 onboarding-automatic_proportions-preparation-next = Ich stehe vor einem Stuhl
 onboarding-automatic_proportions-start_recording-title = Bereiten Sie sich auf ein paar Bewegungen vor
 onboarding-automatic_proportions-start_recording-description = Wir werden nun einige bestimmte Posen und Bewegungen aufnehmen. Diese werden im nächsten Bildschirm angezeigt. Bereiten Sie sicht darauf vor, wenn Sie den Knopf drücken!
 onboarding-automatic_proportions-start_recording-next = Aufnahme starten
 onboarding-automatic_proportions-recording-title = Aufnahme
 onboarding-automatic_proportions-recording-description-p0 = Aufnahme läuft...
 onboarding-automatic_proportions-recording-description-p1 = Machen Sie die unten beschriebenen Bewegungen:
-onboarding-automatic_proportions-recording-steps-0 = Beuge ein paar Mal die Knie.
+onboarding-automatic_proportions-recording-steps-0 = Beugen Sie ein paar Mal die Knie.
```

Comment:
Remember that feets should be planted on floor and not moved.

Reply:
Hm the English text seems not to say, that the feet have to be on the floor. 
So it would mean to make some full squats or is it enough with less?

So for german:
Gehen Sie, mit beiden Füssen am Boden, ein paar mal in die Knie.

Reply:
```suggestion
onboarding-automatic_proportions-recording-steps-0 = Gehen Sie ein paarmal in die Knie.
```
```suggestion
onboarding-automatic_proportions-recording-steps-0 = Gehen Sie, mit beiden Füssen am Boden, ein paarmal in die Knie.
```
```suggestion
onboarding-automatic_proportions-recording-steps-0 = Gehen Sie mehrmals in die Knie.
```
```suggestion
onboarding-automatic_proportions-recording-steps-0 = Gehen Sie, mit beiden Füssen am Boden, mehrmals in die Knie.
```